scripts/scr_ui_manage/scr_ui_manage.gml

```diff
@@ -718,39 +718,47 @@ function scr_ui_manage() {
 	            }
 	            temp3=string(round(ma_health[sel]))+"% HP";temp4="";
 	            // Need abbreviations here
-            
-            
-            
+
+
+
 	            ma_ar="";ma_we1="";ma_we2="";ma_ge="";ma_mb="";ttt=0;
 	            ar_ar=0;ar_we1=0;ar_we2=0;ar_ge=0;ar_mb=0;
-            
+
+							if (ma_armor[sel]!=""){ma_ar=scr_wep_abbreviate(ma_armor[sel]);
+	                // if (string_count("*",ma_ar)>0){ar_ar=2;ma_ar=string_replace(ma_ar,"*","");}
```

Comment:
commented out?

Reply:
The commented sections are inherited from Duke's original code. I'm not entirely sure why they are there in the first place (something about artifacts?), whether they are holdovers from an older version, an internal debug tool he used during development or a remnant from one of his incomplete additions before he abandoned the project.

They can obviously be removed, but I have currently left them in as they do provide some hits as to either how things used to work or possibly regarding intended future functionality and I didn't want to destroy potential documentation for others.